accounts.go

```diff
@@ -0,0 +1,32 @@
+package ledger
+
+import (
+    "fmt"
+    "hash/maphash"
+)
+
+/**
+ * @brief:  Create a new accout for the ledger. The account will be a hash
```

Comment:
```suggestion
 * @brief:  Create a new accout for the ledger. The account will be a hash
  *          of the users full name, date of creation, and the initial balance.
 *
 * @return: The hash value of the account
 **/
func (lgr Ledger) CreateAccountHash(fullname string, initBalance float64) string {
```
Missing comments on the arguments

Reply:
Updated